src/vcu/src/controls.c

```diff
@@ -47,6 +58,7 @@ bool get_controls_enabled(void) {
 
 void execute_controls(void) {
   if (!enabled) return;
```

Comment:
Not paramount, but this function is really long and complicated. Can it be broken up into smaller functions? Also can the logic be structured to use fewer nested if / switch statements / conditionals (&&, !, ||)?

Reply:
Given what we talked about earlier I'm going to resolve this for now

Reply:
(For record's sake: basically we're not gonna try to reorganize this for now but will organize it well for 18)